internal/lsp/commands.go

```diff
@@ -5,21 +5,29 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"strings"
 
+	"github.com/nokia/ntt/internal/fs"
 	"github.com/nokia/ntt/internal/loc"
 	"github.com/nokia/ntt/internal/lsp/protocol"
 )
 
+type param struct {
+	Id  string
+	Uri string
```

Comment:
Abbreviations should be written in all-caps, also some comments describing the fields meaning are helpful:
```suggestion
	ID  string  // Fully qualified testcase identifier
	URI string  // URL points to the ttcn3 source file containing the testcase
```

Note, if applying this suggestion don't forget to update references.